www/Makefile

```diff
@@ -50,6 +50,8 @@ faq-toc.html: faq-content.html ./gentoc.sh
 
 faq.html: faq-toc.html
 
+contrib.html: contrib.txt
+
```

Comment:
This should be taken care of automatically by `TXT = $(shell git ls-files '*.txt')` in line 8, so actually you don't need to (and thus should not) modify the Makefile. (Probably it didn't work for you originally because `contrib.txt` wasn't tracked by git)

Reply:
You're right! I remove that line from there.